INSTALL.md

```diff
@@ -39,6 +39,12 @@ prerequisites:
          libsystemd-dev libusb-1.0-0-dev zlib1g-dev
 
 
+For Fedora you can install these packages:
+
+    sudo dnf install autoconf make automake gcc gcc-c++ kernel-devel avahi-devel \
```

Comment:
@Nopraz I've checked this with SPEC file in Fedora and build log - IIUC kernel-devel is not needed, krb5-devel is needed instead of krb5-libs (krb5-libs will be required implicitly) and libusb1-devel is required instead of libusb-devel (libusb-devel are development files for older version 0.1).

Would you mind updating it?

Reply:
No problem, will do it today